bci_tester/data.py

```diff
@@ -439,6 +455,54 @@ def create_BCI(
     )
 
 
+def create_SPRI(
```

Comment:
why this dupliation of code? is that just for setting `available_versions` ?

Reply:
changed to create_BCI